WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController/NotificationsViewController+Strings.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+extension NotificationsViewController {
+
+    enum Strings {
+        enum NavigationBar {
+            static let notificationSettingsActionTitle = NSLocalizedString(
+                "Notification Settings",
+                comment: "Link to Notification Settings section"
+            )
```

Comment:
Could we please have reverse-DNS keys in these localization strings, to avoid ambiguities in the generated `.strings`? See #19028.

```suggestion
            static let notificationSettingsActionTitle = NSLocalizedString(
                "notificationsViewController.navigationBar.action.settings",
                value: "Notification Settings",
                comment: "Link to Notification Settings section"
            )
```

Reply:
@mokagio I believe these are existing strings that were moved to a different file. Is reverse-DNS keys still the recommendation in such cases?

Reply:
Yes, in that that's the best localization approach we can use, so we should take any occasion we can to introduce it, given it has little overhead.

Thanks 🙇‍♂️

Reply:
Resolved in c9e398106b6415f5a357c9b73fa243854267fd87